definitions/subannual/representative_time_slices.yaml

```diff
@@ -0,0 +1,38 @@
+# description of represnetative periods
+
+- Average week:
+    duration: 168 hours
+    description: average week over a yearly period for the given variable ie. average value of the 52 weekly values of the variable
+
+- Average week|{Season Input}:
+    duration: 168 hours
+    description: average value of the weekly values of the variable, for the weeks belonging to <season>, which are defined as the weeks with at least 4 days belonging to the considered season
+    
+- hhh|Average week:
```

Comment:
What does `hhh` stand for? What is the difference to the category `Average week`?

Reply:
@danielhuppmann , hhh is defined in the description section just below, it is meant to be the indexes of each hour of the week. The difference between hhh|Average week and Average week is that Average week is used for the aggregation over the 168 hours of the week (eg the energy produced in the 'average' week) while hhh|Average week is the energy produced at the 'hhh'th  hour of the average week. There are probably better ways of expressing this but did not find another idea......